src/plugins/range/README.md

```diff
@@ -0,0 +1,71 @@
+- infos = Information about the range plugin is in keys below
+- infos/author = Thomas Waser <thomas.waser@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = presetstorage postgetstorage
+- infos/status = maintained conformant compatible coverage specific unittest tested libc preview experimental unfinished nodoc
+- infos/metadata = check/range check/range/type
+- infos/description = tests if a value is within a given range
+
+## Introduction ##
+
+The range plugin checks if a Key value is within a given range.
+
+## Usage ##
+
+The plugin checks every Key in the Keyset for the Metakey `check/range` which contains either a single range with the syntax `[-]min-[-]max`, or a list of ranges separated by `,` and tests if the Key value is within the range(s).
+
+`check/range/type` can be used to specify the datatype. If not specified otherwise the default value is `INT`
```

Comment:
Please use `type`/`check/type` here, otherwise this is a unnecessary pitfall.

(For hex we need an extra meta data, see discussion  #1388)

Internally, you can use `INT`,.. but please not in the user interface.